### ISSUE TITLE
Jet Tabs - Tabs Widget Header (Menu) Horizontal Scroll on Mobile

### ISSUE THREAD

**Issue by Sayeed1440** [Crocoblock/suggestions#1483]
One thing that frustrates me about tabs is that the menu items of the tab come one by one in mobile devices.

Let say if there are 4 tabs then on mobile they are stacked over each other.

https://prnt.sc/t4is68

Can we make them scroll on mobile devices?

**Comment by realstephenzhao** [comment]
Yes, it's not responsive. See this https://github.com/Crocoblock/suggestions/issues/484

**Comment by stasonua0** [comment]
In addition, please consider enabling anchor links on tabs on your mobile phone. This will enable the user to move to the content part of the tabs when clicking on the tab.
Now I do it like this https://prnt.sc/uduszi, but in the PC version I hide these links.

/*Оригинал*/
В дополнение прошу рассмотреть возможность включения якорных ссылок на вкладках в мобильном телефоне. Это даст возможность пользователю перемещаться в контентую часть табов при клике на таб.
Сейчас это делаю так https://prnt.sc/uduszi , а в ПК версии эти ссылки скрываю.

**Comment by Sayeed1440** [comment]
Hey Guys,

Is there any update?

Do you need any help or something?

Please let me know.

**Comment by Sayeed1440** [comment]
It's been a year now. The issue was opened in June 2020.

It's a serious issue.

**Comment by teaqueue** [comment]
Yeah, this is terrible UX. I have this issue in 2 websites - customer is unhappy. I use a lot of JetPlugins but it's not far off having to use something different for tabs.

**Comment by Lonsdale201** [comment]
If you would like to "enable" the side scroll you can use the following css code:

```
@media screen and (max-width: 767px) {
    .jet-tabs__control-wrapper {
        display: flex;
overflow-x: auto;
overflow-y: hidden;
width: 100%;
white-space: nowrap;
flex-wrap: nowrap!important;
    }
}
```

**Comment by antoinelbch** [comment]
+1

**Comment by esther-bz** [comment]
+1

**Comment by aalexandr21** [comment]
was added

<img width="668" height="762" alt="Image" src="https://github.com/user-attachments/assets/4d3e2944-5401-45a3-9992-f8053cbd0363" />
